utils/build/build.sh

```diff
@@ -26,6 +27,8 @@ while [[ "$#" -gt 0 ]]; do
         -w|--weblog-variant) WEBLOG_VARIANT="$2"; shift ;;
         -e|--extra-docker-args) EXTRA_DOCKER_ARGS="$2"; shift ;;
         -c|--cache-mode) DOCKER_CACHE_MODE="$2"; shift ;;
+        -t|--tracer) TRACER_IMAGE="$2"; shift ;;
+        -tp|--tracer-path) TRACER_LIBRARY_PATH="$2"; shift ;;
```

Comment:
The idea behind this: You can use your own image if you are developing on tracer library and you want to test with your tracer library. For example, when you create a PR of dd-trace-java and system-tests are launched...